src/searchengine/nova3/helpers.py

```diff
@@ -1,4 +1,4 @@
-#VERSION: 1.47
+#VERSION: 1.48
```

Comment:
We should increase appropriate version in master to be greater than the ones from v5.0.

Reply:
I'll handle it in another PR.

Reply:
> I'll handle it in another PR.

Of course. You cannot do it in the same PR anyway.

Reply:
https://github.com/qbittorrent/qBittorrent/pull/21542